core/src/main/java/tc/oc/pgm/core/CoreModule.java

```diff
@@ -118,13 +119,14 @@ public CoreModule parse(MapFactory context, Logger logger, Document doc)
         }
 
         String modes = coreEl.getAttributeValue("modes");
-        List<String> modeList = new ArrayList<>();
+        List<Mode> modeList = new ArrayList<>();
         if (modes != null) {
-          Node node = Node.fromAttr(coreEl, modes);
-          if (node != null) {
-            for (String mode : Splitter.on(" ").split(node.getValue())) {
-              modeList.add(mode);
-            }
+          for (String mode : Splitter.on(" ").split(modes)) {
+            Iterable bigcontext = context.getFeatures().getAll(Mode.class);
+            System.out.println(bigcontext);
+            Mode mode1 = context.getFeatures().get(mode, Mode.class);
+            modeList.add(mode1);
+            System.out.println(mode1);
```

Comment:
You left out some debugs here, also, you need to check that the mode returned isn't null or else it means that mode hasn't been found, again, you have a perfect 1 to 1 example in https://github.com/PGMDev/PGM/blob/dev/core/src/main/java/tc/oc/pgm/flag/FlagParser.java#L105